src/Features/VisualBasic/Portable/EditAndContinue/VisualBasicEditAndContinueAnalyzer.vb

```diff
@@ -2612,6 +2612,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.EditAndContinue
 
 #Region "State Machines"
 
+        Protected Overrides ReadOnly Property SupportsStateMachineUpdates As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        Protected Overrides Function IsStateMachineResumableStateSyntax(node As SyntaxNode) As Boolean
+            Return node.IsKind(SyntaxKind.AwaitExpression, SyntaxKind.YieldStatement)
```

Comment:
I was surprised to see this doing anything in VB, so I looked for where this method is called, and it isn't. Coming soon?

Reply:
Yes, coming soon.

Reply:
Actually, will be removed for both in coming PR.